tests/installation/ipxe_install.pm

```diff
@@ -95,12 +99,26 @@ sub set_bootscript {
     $cmdline_extra .= " root=/dev/ram0 initrd=initrd textmode=1" if check_var('IPXE_UEFI', '1');
 
     if ($autoyast ne '') {
-        $cmdline_extra .= " autoyast=$autoyast ";
+	    #julie        $cmdline_extra .= " autoyast=$autoyast ";
+        $cmdline_extra .= " autoyast=$autoyast sshd=1 sshpassword=$testapi::password ";
```

Comment:
During autoyast installation, no interactive is required, I think we need not sshd and sshpassword for boot cmdline, how about you?

Reply:
seems to depend on the trade-off between the potential security concerns of including passwords in the command line and the practical benefits of having SSH access for troubleshooting purposes during non-interactive AutoYaST installations. What happens if we remove it => A hook might need SSH access to collect important logs, I hope the understanding is correct.

Reply:
Yes, you get it right. This secrect of ssh login is not so sensible as the IPMI access.

Reply:
Using ssh allows user to login anyway. I think we should keep it.

Reply:
https://en.opensuse.org/SDB:Linuxrc
```
ssh=1  --  This parameter enables access to linuxrc via SSH when performing the installation with YaST in text mode or via X11 forwarding. Use ssh -X root@hostname or ssh -Y root@hostname for X11 forwarding.

sshd=1  --- This parameter enables login to the installation system while the installation is running.
```

Reply:
Yes. I was wondering whether ```ssh=1``` works with auotyast.

Reply:
Not sure. Let's see in future tests.  'ssh=1' is not propriate, because some tests also run on sol console or on VNC graphic mode.